docs/user-guide-device-management.md

```diff
@@ -1,25 +1,173 @@
 
-## Device Registration
+## Devices
 
-To register a device, enter the device details in the registration form. You are asked to input name, technical data and location data about the device. You can choose to upload device images and add a description about the project in the Project Story.
+‘Devices’ are electricity generating assets (e.g solar pv, hydroelectric dam, steam turbine.) They must be registered in the system to provide their capacity, location and generation data. This data is used to provide trust in the validity of the certificates that are issued for their generation. A user that is associated with a registered [organization](./user-guide-reg-onboarding.md#organizations) can [register a device](./user-guide-reg-onboarding.md#registering-devices) within the system if they have the necessary role permissions. Read more about role permissioning [here](./user-guide-reg-onboarding.md#user-roles-and-hierarchy). 
+
+### Accessing Device Generation Data
 
 Origin offers the possibility to connect an external metering system. In order for this to work, a custom integration to the specific metering system is required. Devices have to be approved by the issuer in order to be used on the platform.
 
-The local issuer receives the device registration and can verify the data and approve the device. Everything that involves additional processes of the registry, e.g. if there is a need for an on-site visit or additional documents, is handled directly between the user and the registry.
+The local issuer receives the device registration and can verify the data and approve the device. Everything that involves additional processes of the registry, (e.g. if there is a need for an on-site visit or additional documents), is handled directly between the user and the registry.
+
+Once you have an active account in the Origin platform and are part of an organization, you can register your devices on the platform.
+
+### Managing Devices on the Origin Platform
+
+The Device interface has four views. The purpose and utility of each view is explained in detail below.
+
+![Device Interface](images/deviceMgmtImages/DevicePortalAll.png)
+
+1. <b>[All Devices:](##all-devices)</b> Allows any user to see all registered devices within a marketplace. Selecting a single device allows you to see the detail view of each device. You do not need to be logged in to view this interface.
+2. <b>[Map View:](##map-view)</b> Allows any user to see registered devices within a marketplace as locations on a map. Selecting a single device allows you to see the detail view of each device. You do not need to be logged in to view this interface.
+3. <b>[My Devices:](##my-devices)</b> Allows any logged in user to see their organization’s registered devices and each device’s status.
+4. <b>[Register Device:](##register-device)</b> Allows any logged in user that is registered with an organization to register a device on behalf of that organization.
+
```

Comment:
Those are devices which are linked to user's I-REC account but do not exist yet in Origin system. 
There are lots of them because it is test environment. They will appear under "My Devices" after you import them

Reply:
"Import Device" page can become available only after I-REC connection is set. 
And only for certain I-REC account types. (not remember which ones exactly)

Reply:
That makes sense. So once the I-REC connection is set - are all the devices in "Devices to Import" automatically populated through the I-REC API?